students/mathewcm/break_me.py

```diff
@@ -8,7 +8,7 @@
 #    if a > b
 
 def test4():
-   raise AttributeError
+   raise AttributeErrorg
```

Comment:
The idea was to write some code that would raise an AttributeError, rather than raising it yourself :-)

We’ll cover raising and handling Exceptions in a couple weeks